app/services/katello/pulp3/content_view_version/import_validator.rb

```diff
@@ -57,6 +58,14 @@ def ensure_from_cvv_exists!
           end
         end
 
+        def ensure_manifest_imported!
+          rh_repos = ::Katello::Pulp3::ContentViewVersion::Import.metadata_map(metadata, redhat_only: true)
+          if rh_repos.any? || !content_view.organization.manifest_imported?
```

Comment:
Should be an and here not a or.  

```suggestion
          if rh_repos.any? && !content_view.organization.manifest_imported?
```